projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -959,7 +959,14 @@ private InterfaceSet computeFlowSinks(
             }
           }
           if (bestCandidates.size() != 1) {
-            throw new BatfishException("multiple best vrrp candidates:" + bestCandidates);
+            String deterministicBestCandidate = bestCandidates.first();
+            _logger.redflag(
+                "Arbitrarily choosing best vrrp candidate: '"
+                    + deterministicBestCandidate
+                    + " for prefix/groupNumber: '"
+                    + p.toString()
```

Comment:
toString unnecessary (esp in String.format)

Reply:
I use toString() to force NPE if Prefix was null somehow